library/public/options.h

```diff
@@ -108,6 +108,16 @@ class F3D_EXPORT options
    */
   std::vector<std::string> getNames();
 
+  /**
+   * Utility static function used to compute the Levenshtein distance between two strings.
+   */
+  static unsigned int textDistance(const std::string& strA, const std::string& strB);
```

Comment:
Im not sure this should be here, but I have no better suggestion

Reply:
we probably need a `f3d::utils::` with static functions but it would be weird with a single function.

Reply:
The problam is that it is in the public API. Once we release this, its hard to remove it from options without breaking other people code. I do think we need a f3d::utils, lets create it ?

Reply:
done